api/src/pcapi/connectors/acceslibre.py

```diff
@@ -263,11 +263,52 @@ def extract_street_name(address: str | None = None, city: str | None = None, pos
     return ""
 
 
+def address_match(
+    acceslibre_address: str | None = None,
+    acceslibre_city: str | None = None,
+    acceslibre_postal_code: str | None = None,
+    acceslibre_ban_id: str | None = None,
+    venue_address: str | None = None,
+    venue_city: str | None = None,
+    venue_postal_code: str | None = None,
+    venue_ban_id: str | None = None,
+) -> bool:
+    # If banIds are matching, we return True
+    ban_ids_match = venue_ban_id and venue_ban_id == acceslibre_ban_id
+    # If cities / postal codes are given, we must check that they do not mismatching, else we return False
+    cities_mismatch = (venue_city and venue_city != acceslibre_city) or (
+        venue_postal_code and venue_postal_code != acceslibre_postal_code
+    )
```

Comment:
[détail / performance] Ici aussi, en sortant prématurément, on peut s'éviter l'appel à `fuzz.ratio()` si la ville ou le code postal ne correspond pas.

```suggestion
    # J'en ai profité pour reformuler/simplifier le commentaire.
    # If cities / postal codes are given, they must match.
    if (
        (venue_city and venue_city != acceslibre_city)
        or (venue_postal_code and venue_postal_code != acceslibre_postal_code)
    ):
        return False
```

Avec cette forme et la sortie prématurée suggérée dans le commentaire précédent, on peut alors simplifier la sortie finale avec : 

```python
return addresses_match
```